packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -7,6 +7,7 @@ import { get, escapeRegExp } from 'lodash';
 
 let language = 'en-US';
 let window = null;
+let stub = null;
```

Comment:
I tried adding stub as an alias instead. But it wasn't working correctly so a global stub variable seems OK in this instance.